client/js/views/GameView.vue

```diff
@@ -360,21 +327,23 @@
         @targetedOneOff="beginTargeting($event)"
         @cancel="clearSelection"
       />
+      <game-dialogs
+        :resolving-seven="resolvingSeven"
```

Comment:
Any particular reason to prefer a prop to mapping the store getter inside the child?

Reply:
Nope, just missed it in my conversion